app/presenters/money_question_presenter.rb

```diff
@@ -6,7 +6,7 @@ def response_label(value)
   end
 
   def hint_text
-    text = [body, suffix_label].reject(&:blank?).compact.join(", ")
+    text = [body, hint, suffix_label].reject(&:blank?).compact.join(", ")
```

Comment:
Is it worth writing a test for this?

Reply:
good point, mainly for documentation I've added a couple of specs for this (and another very similar presenter)